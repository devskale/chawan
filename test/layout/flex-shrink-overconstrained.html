<!DOCTYPE html>
<div style="display: flex; width: 40ch">
<div style="width: 40ch; background: red">1111 1111 1111 1111 1111 1111 1111 1111 1111</div>
<div style="width: 80ch; background: pink">test test test test test test test test test test test test test test test test test</div>
<!--
<div style="width: 4ch; background: aliceblue">last</div>
