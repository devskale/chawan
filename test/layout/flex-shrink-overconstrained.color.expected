[48;2;255;0;0m1111 1111    [38;2;95;95;95m[48;2;255;192;203mtest test test test test   [39m[49m
[48;2;255;0;0m1111 1111    [38;2;95;95;95m[48;2;255;192;203mtest test test test test   [39m[49m
[48;2;255;0;0m1111 1111    [38;2;95;95;95m[48;2;255;192;203mtest test test test test   [39m[49m
[48;2;255;0;0m1111 1111    [38;2;95;95;95m[48;2;255;192;203mtest test                  [39m[49m
[48;2;255;0;0m1111         [38;2;95;95;95m[48;2;255;192;203m                           [39m[49m
